src/onboarding/github.md

```diff
@@ -0,0 +1,61 @@
+# GitHub Tickets
+
+GitHub is a service that provides code hosting and versioning. We also use it for its collaborative features, such as issue/bug tracking and project boards. This documents explains how to use GitHub to keep track of tasks.
+
+## Repositories
+
+Here is the homepage for the [toronto-community-network](https://github.com/tomeshnet/toronto-community-network) repository.
```

Comment:
The text is a little akward
`Here is the homepage for the toronto-community-network repository.`
Unless you include the link the language does not make sense. 

Also is it a homepage?

Maybe

```suggestion
The main repository for the Toronto Community Network can be found on github under [tomeshenet/toronto-community-network](https://github.com/tomeshnet/toronto-community-network).
```